docs/rules/object-key-newlines.md

```diff
@@ -0,0 +1,68 @@
+# Enforce placing object keys on separate lines (object-key-newlines)
+
+While formatting preferences are very personal, a number of style guides require that
```

Comment:
@nzakas Correction the previous comment after searching for newline in existing rule identifiers:
object-property-newline (singular instead of plural)